src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,15 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import { User } from '../../types/User';
+
+interface UserInfoProps {
```

Comment:
use simple names for props interfaces
```suggestion
interface Props {
```

Reply:
Done